pkg/nrtupdater/nrtupdater.go

```diff
@@ -126,7 +126,9 @@ func (te *NRTUpdater) Run(infoChannel <-chan MonitorInfo) chan<- struct{} {
 				}
 				tsEnd := time.Now()
 
-				log.Printf("update request received at %v completed in %v", tsBegin, tsEnd.Sub(tsBegin))
+				tsDiff := tsEnd.Sub(tsBegin)
+				prometheus.UpdateOperationDelayMetric(float64(tsDiff.Milliseconds()))
```

Comment:
we need to do the same in `pkg/resourcetopologyexporter/resourcetopologyexporter.go:199`: